test/extensions/filters/http/alternate_protocols_cache/filter_integration_test.cc

```diff
@@ -154,7 +154,7 @@ class MixedUpstreamIntegrationTest : public FilterIntegrationTest {
     size_t seconds = std::chrono::duration_cast<std::chrono::seconds>(
                          timeSystem().monotonicTime().time_since_epoch())
                          .count();
-    std::string value = absl::StrCat("h3=\":", port, "\"; ma=", 86400 + seconds);
+    std::string value = absl::StrCat("h3=\":", port, "\"; ma=", 86400 + seconds, "|0");
```

Comment:
Can one of the tests verify alt_svc_cache.txt has srtt data saved in the end?

Reply:
good call - will take some test utility work
/wait